sgt-back-end/index.js

```diff
@@ -0,0 +1,168 @@
+const express = require('express');
+const app = express();
+const pg = require('pg');
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+  select *
+  from "grades";
+  `;
```

Comment:
Despite what the example shows, in general, please `select` specific column names to avoid leaking sensitive information from your databases.